### PR TITLE
Change up PR template

### PR DESCRIPTION
**Pull request by shrielenee**
## Scope of this PR

- __Editorial__: Typos, grammatical inconsistencies, or minor rewrites

## Purpose of this PR

With the improvements to the PR form, Don and I felt like it could be better. I'm suggesting this simple list style to make it easier to read and see what needs to be done.

## Affected URLs

- https://github.com/magento/devdocs/blob/master/.github/PULL_REQUEST_TEMPLATE.md

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,14 +1,17 @@
 ## Scope of this PR
 
-<!-- REQUIRED We set [labels](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html#devdocs) based on the scope of changes with a PR. -->
+<!-- REQUIRED
+1. Leave the label that best describes the scope of your PR visible.
```
Comment (hguthrie) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
1. Choose one of the following labels that best describes the scope of your PR and do not delete it.
```

Comment (shrielenee) [reply .github/PULL_REQUEST_TEMPLATE.md]: TBH this seems longer and less clear. I like the specificity of it, but we're going for a quick understanding of what's needed.

Comment (dobooth) [reply .github/PULL_REQUEST_TEMPLATE.md]: I would rather explicitly tell them to also delete the others rather than imply it.

**Comment by shrielenee** [comment]
running tests

**Comment by dobooth** [comment]
running tests

**Pull request merged**